### ISSUE TITLE
[Query] Helping user with key binding cues.

### ISSUE THREAD

**Issue by piyush-das** [kazhala/InquirerPy#7]
Hi, 

Besides the prompt message, is a there a way to give cues about the key bindings (like some sub text which has a greyed out styling )? This would be particularly useful in the case where there is a multi-select or a single selection available. Additionally it might also inform the user that using tab would help to kick in the auto-completion.

Or perhaps this information section could also be given to indicate additional information which might help the user in answering or making selections.

Thanks

**Comment by kazhala** [comment]
Hi @piyush-das ,

At the moment, you can leverage the parameter ```instruction``` to provide some keybinding information to the user.

```python
result = inquirer.select(
    message="Select one:", choices=[1, 2, 3], default=2, instruction="ctrl-p, ctrl-n to navigate"
).execute()
```

The instruction will be printed using a different colour behind the question. You can also customise the instruction style. It is documented [here](https://github.com/kazhala/InquirerPy/wiki/Style).

It is not currently supported for none list type prompts, which I think probably should...I'll add that to next release. It probably should also have an option to include the instructions in the second line or a dedicated area?

**Comment by piyush-das** [comment]
Yeah , the concept of customization instructions to prompts seems like a great addition. Thanks

**Comment by kazhala** [comment]
The latest release 0.3.0 now includes the parameter `long_instructions" which you can use to print longer instructions such as keybindings which will be displayed at the bottom of the screen.
